packages/firestore/src/api/database.ts

```diff
@@ -424,6 +427,36 @@ export function setLogLevel(level: PublicLogLevel): void {
   setClientLogLevel(level);
 }
 
+export function loadBundle(
+  db: Firestore,
+  bundleData: ArrayBuffer | ReadableStream<Uint8Array> | string
+): ApiLoadBundleTask {
+  const resultTask = new LoadBundleTask();
```

Comment:
We need to figure out where to call "ensureClientConfigured". It might depend on how we expose this function in the end.